Scripts/create-archive.sh

```diff
@@ -7,18 +7,20 @@ set -e
 PROJECT_DIR="$(dirname "$0")/.."
 
 # Create temporary directory
-TEMP_DIR=$(mktemp -d -t "$1") 
-mkdir -p "$TEMP_DIR/$1"
+TEMP_ARCH_NAME=`echo $1 | cut -d'-' -f 1`
```

Comment:
Why do we need "| cut -d'-' -f 1`" here. It wasn't there before the changes.

Reply:
to avoid hardcoding of file name now the name will be got from result-name parameter without version

Reply:
Let's use $PRODUCT_NAME and fallback to "| cut -d'-' -f 1`" if it is not defined?